<launch> 
    <node pkg="assignment1" type="apriltag_locator_service" name="apriltag_locator_service"/>
    <node pkg="assignment1" type="movement_handler" name="movement_handler"/>
    <node pkg="assignment1" type="map_waypoints_service" name="map_waypoints_service"/>
    <node pkg="assignment1" type="node_A" name="node_A"/>
</launch>
